### PR TITLE
Adding test for module unload/load[V3]

### PR DESCRIPTION
**Pull request by hathyaga**
This Program loads and unloads the kernel driver modules along with the dependant modules for a total of 100 counts as a stress test.

### CODE REVIEW

**Comment by hathyaga** [comment]
@idoktor Here is the new PR containing all the changes suggested.

**Comment by hathyaga** [comment]
[job.log.tar.gz](https://github.com/avocado-framework/avocado-misc-tests/files/434534/job.log.tar.gz)

File: io/driver/module_unload_load.sh.data/README.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This Program loads and unloads the kernel driver modules. For the kernel driver modules that have dependencies, the dependant modules should  be listed in the config file (config). The script takescare of unloading the dependant modules, if any, before unloading the core module. It runs for a total of 100 times as stress.
+
```
Comment (ldoktor) [inline io/driver/module_unload_load.sh.data/README.txt:2]: You should also mention, that this test passes when at least one of the drivers finishes the whole loop.

Additionally it'd be worth mentioning that it picks only modules listed in `lspci`.

Apart from those little clarifications I think it's ready for inclusion.

**Comment by apahim** [comment]
@hathyaga, this repository is frozen. Please send the PR with the requested changes to:
https://github.com/avocado-framework-tests/avocado-misc-tests

**Pull request closed without merging**